Makefile

```diff
@@ -1,6 +1,6 @@
+GIT_TAG := $(shell git tag --contains)
+RELEASE_TAG := $(shell [[ $(GIT_TAG) =~ ^.*rc.* ]] && echo 'prerelease' || echo 'latest')
 
-build:
-	yarn install && yarn build
-	mkdir -p lib
-	sed  's|"lib/index.js"|"index.js":|g' package.json > lib/package.json
-	rm -rf node_modules/
+.PHONY: publish
+publish:
+	npm publish --access public --tag $(RELEASE_TAG)
```

Comment:
Does this need to bomb out if no tag is available?

Reply:
If there's no `GIT_TAG` then it will default to `latest`, which is also the default when doing `npm publish` with no parameters. 

It's not something I think we need to worry about, because this target is only called by CI, and the CI job only runs when a git tag is created that starts with `v`